service/storage_service.cc

```diff
@@ -3178,6 +3197,10 @@ future<> storage_service::replicate_to_all_cores(mutable_token_metadata_ptr tmpt
                 it = table_erms.erase(it);
             }
 
+            if (!view_erms.empty()) {
+                throw std::runtime_error(fmt::format("Found orphaned pending effective_replication_maps for the following views: {}", boost::adaptors::keys(view_erms)));
```

Comment:
Please don't add more boost stuffs

Reply:
Fixed in v4 ([cf0d00a](https://github.com/scylladb/scylladb/commit/cf0d00ac904a20fb770ff85a4e7da11db99130b2)):